probe/endpoint/dns_snooper.go

```diff
@@ -250,15 +250,14 @@ func (s *DNSSnooper) processDNSMessage(dns *layers.DNS) {
 		domainQueried = question.Name
 		records       = append(dns.Answers, dns.Additionals...)
 		ips           = map[string]struct{}{}
-		alias         []byte
+		aliases       = [][]byte{}
```

Comment:
Would this work better as a `map` ? I suspect it depends on how many aliases we get.

Reply:
I can rewrite it as a map, no problem.

Reply:
`[]byte` is not allowed as a map key - it would have to be `string`.
Yes, the relative expense of map vs linear search depends on how many elements you have.
From [Rob Pike's 5 Rules of Programming](https://users.ece.utexas.edu/~adnan/pike.html) "n is usually small"
However I was thinking more about code readability than performance.  It's not a big deal.